packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -104,11 +128,24 @@ describe('HCP home page', () => {
     expect(removeButton).toBeVisible()
 
     await userEvent.click(removeButton)
+
     const removeDialog = screen.getByRole('dialog')
     expect(removeDialog).toBeVisible()
+
+    const title = within(removeDialog).getByText(`Remove ${unmonitoredPatient.profile.fullName} from ${teamThree.name}`)
+    expect(title).toBeVisible()
+    const question = within(removeDialog).getByTestId('modal-remove-patient-question')
+    expect(question).toHaveTextContent(`Are you sure you want to remove ${unmonitoredPatient.profile.fullName} from ${teamThree.name}?`)
+    const info = within(removeDialog).getByText('You and the care team will no longer have access to their data.')
+    expect(info).toBeVisible()
+    const alertInfo = within(removeDialog).getByText('If you want to remove the patient from another care team, you must first select the care team from the dropdown menu at the top right of YourLoops.')
+    expect(alertInfo).toBeVisible()
+    const cancelButton = within(removeDialog).getByText('Cancel')
```

Comment:
Same question

Reply:
The click on `confirmRemoveButton` is done in this test, see lines 146-157

Reply:
I am but a blind boi :dark_sunglasses:

Reply:
:man_with_probing_cane: